### PR TITLE
data: EmaniAditya

### PR DESCRIPTION
**Pull request by EmaniAditya**
Hello there, My name is Aditya Reddy. I am creating a new account on LinkFree.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

image

## Note to reviewers

I have checked my json and it's a valid one.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7202"></a>

### CODE REVIEW

File: data/EmaniAditya.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Aditya Reddy",
+  "displayStatusPublic": true,
+  "type": "personal",
+  "bio": "trying to Escape the Matrix. </3 . 🤫 !",
+  "avatar": "https://github.com/EmaniAditya.png",
```
Comment (kumarsonsoff3) [inline data/EmaniAditya.json:6]: Please remove `avatar`, `type`, & `displayStatsPublic` keys from your `json` as we no longer need them. 

```suggestion
  "bio": "trying to Escape the Matrix. </3 . 🤫 !",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/EmaniAditya.json (hunk 1 above)
Comment (EmaniAditya) [reply data/EmaniAditya.json]: Made the required changes.
image

**Review by EmaniAditya — COMMENTED** [review]
Made the required changes.
image

**Review by harshsingh32 — COMMENTED** [review]
Looks good

**Review by VulcanWM — APPROVED** [review]
looks great!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**